Assets/Scripts/chooseCharacter/chooseCharacter.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Beweegmaatje
+{
+    /* <summary>
+     * In this class you can choose your character
+    </summary>*/
+    public class ChooseCharacter : MonoBehaviour
+    {
+
+        [SerializeField]
+        private GameObject _character;
+
+        public Material[] characterColor;
+
+        private int _colorNumber = 0;
+
+        // Start is called before the first frame update
+        void Start()
+        {
+           
+        }
+
+        // Update is called once per frame
+        void Update()
+        {
+        
+        }
```

Comment:
Remove empty functions from your scripts to reduce the amount of 'zombie code' being left behind